docs/sections/cn_workflows.md

```diff
@@ -359,6 +359,55 @@ popper run -f sample.yml -c config.yml
 
 This runs the step `one` locally in the host and step `two` through slurm on 2 nodes.
 
+## Kubernetes
+
+Popper can be run [Kubernetes cluster](https://kubernetes.io/docs/setup/) as the resource manager to orchestrate the execution of a step to one a single node.
```

Comment:
This sentence needs a bit of work. Maybe something like "Popper can execute a workflow using a Kubernetes cluster as the underlying resource manager".

Reply:
Nice! This way I don't think I will need to include _orchestrate the execution of a step to run a single node_. Or do you think it is important to include it?